pkg/antctl/antctl.go

```diff
@@ -538,6 +539,12 @@ var CommandList = &commandList{
 			supportController: false,
 			commandGroup:      mc,
 		},
+		{
+			cobraCommand:          update.UpdateCmd,
+			supportAgent:          true,
+			supportController:     true,
```

Comment:
are these intentional? I just want to make sure you have given this some thought. The PR description doesn't mention from where antctl can be run to modify the FA configuration parameters.
Now that I see this, I understand why you modified RBAC permissions for both the Agent and the Controller. However I think you have to consider the following:
- does it make sense to run these commands from the Agent Pod, is it something we want to support?
- similarly, does it make sense to run these commands from the Controller Pod (this one can be a bit tricky because we also use `ModeController` for "out-of-cluster" operations)?
- if you only support this command for `ModeFlowAggregator` (i.e. from the Flow Aggregator Pod), then you can simplify the logic to determine the ConfigMap's name, by leveraging the Pod downward API: https://github.com/antrea-io/antrea/blob/7a793898ff70156736eddd1ebe2b5a93d1222c25/build/yamls/antrea.yml#L3152-L3153
- if you want to support `ModeController`, you need a less generic command name

Reply:
I was thinking it might be convenient if we can mutate fa configmap from out-of-cluster.
After I discussed with @dreamtalen , we will only support "antctl set" command in FA pod.